data/projects/create_jenkins_plugin.md

```diff
@@ -1,26 +1,45 @@
-Jenkins users should be able to see coala results right in the 
-Jenkins UI. The result of this project should be a full fledged Jenkins 
-integration, living in an own repository with a full testsuite.
+Jenkins is the most popular ci-cd server, and there are many corporations 
+who already use it. Integrating coala into Jenkins can attract business 
+clients better who could end up sponsoring/helping out coala. It also generally allows a wider range of users to use coala in their prefered way.
```

Comment:
In my opinion, now we could use coala in jenkins by writing shell commands to install and run coala, and see result in build logs.

And this plugin allows users to run coala from jenkins UI, and see the result just like warnings plugin.

I get the idea from docker plugin. Users could run docker command from shell without this plugin but if they install this plugin, the process is clear.

<img width="265" alt="2017-02-07 8 18 34" src="https://cloud.githubusercontent.com/assets/5100735/22690875/b4c55ab0-ed72-11e6-85d3-794c258407f8.png">
<img width="298" alt="2017-02-07 8 19 03" src="https://cloud.githubusercontent.com/assets/5100735/22690874/b4c4cb36-ed72-11e6-869b-b2228c516c91.png">